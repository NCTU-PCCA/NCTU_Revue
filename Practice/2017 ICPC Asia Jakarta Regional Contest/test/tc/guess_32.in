591723034310272895 7
><><<>>
