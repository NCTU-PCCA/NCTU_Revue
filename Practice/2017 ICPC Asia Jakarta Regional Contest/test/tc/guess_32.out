1 591723034310272895 2 591723034310272894 591723034310272893 3 4
